lighthouse-core/gather/driver.js

```diff
@@ -46,11 +46,17 @@ class Driver {
     this._devtoolsLog = new DevtoolsLog(/^(Page|Network)\./);
     connection.on('notification', event => {
       this._devtoolsLog.record(event);
-      this.recordNetworkEvent(event.method, event.params);
+      this._recordNetworkEvent(event.method, event.params);
       this._eventEmitter.emit(event.method, event.params);
     });
     this.online = true;
     this._domainEnabledCounts = new Map();
+
+    /**
+     * Used for monitoring network status events.
+     * @private {?NetworkRecorder}
+     */
+    this._networkStatus = null;
```

Comment:
yeah, I just wanted to move away from the "records" part since we aren't using it for records. We're using it for `busy`, `idle`, and `requestloaded` events. Not sure what to call that :)